assets/js/application.js

```diff
@@ -1,2 +1,4 @@
+//= require google-analytics.js
+//= require jquery-3.2.1.slim.min.js
```

Comment:
Our new `Content-Security-Policy` forbids inline `<script>`s from running. We therefore embed the GA code in the main compiled JavaScript file.